tcp/server.go

```diff
@@ -49,35 +57,36 @@ func getTCProot(root string) string {
 	return root
 }
 
-//log Error if one occures
-func checkErr(err error) {
-	if err != nil {
-		qbin.Log.Errorf("TCP Server Error: %s", err.Error())
-	}
-}
-
 //handle incomming TCP Connection
 func handleClient(connTCP net.Conn, root string) {
+	qbin.Log.Debug("Handeling incomming connection...")
 	//set timeout to 5 seconds, limit requestlength to 1MB, close connection on programm close
-	connTCP.SetReadDeadline(time.Now().Add(5 * time.Second))
+	connTCP.SetReadDeadline(time.Now().Add(1500 * time.Millisecond))
 	result := make([]byte, 1024)
 	defer connTCP.Close()
 
 	result, err := ioutil.ReadAll(connTCP)
-	checkErr(err)
+	if err != nil {
+		qbin.Log.Errorf("TCP-read error: %s", err)
```

Comment:
This is also called for `TCP-read error: read tcp 127.0.0.1:9000->127.0.0.1:45668: i/o timeout` which is basically the error thrown when SetReadDeadline is reached, which should lead to a success instead of an error.